runtime.c

```diff
@@ -385,6 +392,38 @@ main (int argc, char *argv[])
         exit(0);
     }
 
+    if (arg && strcmp(arg,"appimage-portable-home")==0) {
+        char portable_home[2048];
+        int mkdir_ret;
+
+        sprintf(portable_home, "%s.home", appimage_path);
+        mkdir_ret = mkdir(portable_home, S_IRWXU);
```

Comment:
Could you replace this with a `switch`-`case` construct?

Reply:
Even though I'd normally agree with using switch-case instead of `if-else if-else`, in this case a 6 line piece of code just becomes 11 lines

Reply:
Well, I intended to get rid of the `mkdir_ret` variable. With `switch`, you can embed the `mkdir` call into the brackets, removing another two lines.